sdk/dotnet/Network/Inputs/VpnGatewayBgpSettingsArgs.cs

```diff
@@ -10,7 +10,7 @@
 namespace Pulumi.Azure.Network.Inputs
 {
 
-    public sealed class VpnGatewayBgpSettingArgs : Pulumi.ResourceArgs
+    public sealed class VpnGatewayBgpSettingsArgs : Pulumi.ResourceArgs
```

Comment:
What happened to this name? It's a breaking change.

Reply:
They *added* MaxItems: 1` here :/ https://github.com/terraform-providers/terraform-provider-azurerm/pull/9035/files#diff-a755240392b1658fa6f3ba7a1af73a46b17e2c07139cd248fd161195ac2ff8ecR64